research/a3c_blogpost/a3c_cartpole.py

```diff
@@ -347,15 +347,13 @@ def compute_loss(self,
     value_loss = advantage ** 2
 
     # Calculate our policy loss
-    actions_one_hot = tf.one_hot(memory.actions, self.action_size, dtype=tf.float32)
-
     policy = tf.nn.softmax(logits)
-    entropy = tf.reduce_sum(policy * tf.log(policy + 1e-20), axis=1)
+    entropy = tf.nn.softmax_cross_entropy_with_logits_v2(labels=policy, logits=logits)
 
-    policy_loss = tf.nn.softmax_cross_entropy_with_logits_v2(labels=actions_one_hot,
-                                                             logits=logits)
+    policy_loss = tf.nn.sparse_softmax_cross_entropy_with_logits(labels=memory.actions,
+                                                                 logits=logits)
     policy_loss *= tf.stop_gradient(advantage)
-    policy_loss -= 0.01 * entropy
+    policy_loss = 0.01 * entropy
```

Comment:
I think "policy_loss -= 0.01 * entropy" might be correct, overwriting the policy_loss does not make sense.

Reply:
You are totally correct. I fixed this before merging.